app/stacks/uk-south/applications-service/app-service.tf

```diff
@@ -10,8 +10,7 @@ module "app_services" {
   app_service_private_dns_zone_id                                 = data.azurerm_private_dns_zone.app_service.id
   applications_service_public_url                                 = var.applications_service_public_url
   applications_service_redis_connection_string_secret_name        = azurerm_key_vault_secret.redis_cache_connection_string[count.index].name
-  applications_sql_server_connection_string                       = local.sql_connection_string
-  applications_sql_server_connection_string_jbdc                  = local.sql_jbdc_connection_string
+  applications_sql_server_connection_string                       = azurerm_key_vault_secret.applications_sql_server_connection_string
```

Comment:
I think this variable can be removed. As above, I don't think the `applications-app-services` component is actually using it.

Reply:
Will remove but do you mind explaining please? Isn't this used in our application service?